### DISCUSSION TITLE
Dynamic custom model.

### DISCUSSION THREAD

**Discussion by phungleson** [SeaQL/sea-orm#1108] [General]
I would like to dynamically select some columns into a model.

My model looks like this.

```
#[derive(FromQueryResult)]
pub struct CustomModel {
    pub id: Option<Uuid>,
    pub name: Option<String>,
    pub description: Option<String>,
}
```

So if I select `id` it should return `Some(id)`, the rest are `None`.

Would this work with current implementation?

Additionally, I would like to have a field in `CustomModel` where I would like `sea-orm` to ignore. Something like this (but it seems to work only on Model atm)

```
#[derive(FromQueryResult)]
pub struct CustomModel {
   ...
    #[sea_orm(ignore)]
    pub ignore_me: Option<String>,
}
```

Thanks!

**Comment by billy1624** [comment]
Hey @phungleson, custom model with `Option<T>` field is possible, see [here](https://github.com/SeaQL/sea-orm/blob/0f8b4bd1fa5856e961f39bebf954530a814dbd49/tests/relational_tests.rs#L55-L59) for example.

**Reply by phungleson** [reply]
Thanks!
